packages/transactions/src/builders.ts

```diff
@@ -137,6 +141,71 @@ export async function estimateTransfer(
   return feeRate * txBytes;
 }
 
+interface FeeEstimation {
+  fee: bigint,
```

Comment:
Are these `bigint` or `number`? The unit tests have `number`s

Reply:
changing them to `number`